src/main/java/gregtech/loaders/MaterialInfoLoader.java

```diff
@@ -13,281 +14,369 @@
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[3].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.StainlessSteel, M * 8), // plate
-                new MaterialStack(Materials.Gold, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[4].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Titanium, M * 8), // plate
-                new MaterialStack(Materials.Aluminium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[5].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.TungstenSteel, M * 8), // plate
-                new MaterialStack(Materials.Platinum, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[6].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.RhodiumPlatedPalladium, M * 8), // plate
-                new MaterialStack(Materials.NiobiumTitanium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[7].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.NaquadahAlloy, M * 8), // plate
-                new MaterialStack(Materials.VanadiumGallium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[8].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Darmstadtium, M * 8), // plate
-                new MaterialStack(Materials.YttriumBariumCuprate, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-        OreDictUnifier.registerOre(MetaTileEntities.HULL[9].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.Neutronium, M * 8), // plate
-                new MaterialStack(Materials.Europium, M), // single cable
-                new MaterialStack(Materials.Rubber, M * 2))); // plate
-
-
-        OreDictUnifier.registerOre(MetaTileEntities.ENERGY_INPUT_HATCH[3].getStackForm(), new ItemMaterialInfo(
-                new MaterialStack(Materials.StainlessSteel, M * 8), // plate
-                new MaterialStack(Materials.Gold, M * 2), // single cable
-                new MaterialStack(Materials.Rubber, M * 4), // plate
-                new MaterialStack(Materials.BlackSteel, M * 2), // fine wire
-                new MaterialStack(Materials.SteelMagnetic, M / 2) // rod
+        // Hull Composition is the Assembler recipe, minus the fluid input
+        // Use Silicone Rubber for cable coverings when both Silicone Rubber and SBR can cover cables
```

Comment:
I'm not sure if we want this. It allows transmuting SBR into SiR which I don't think is desired.